clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -742,6 +744,14 @@ private void destroyServerStore(ClientDescriptor clientDescriptor, DestroyServer
     boolean isDuplicate = isLifeCycleMessageDuplicate(destroyServerStore);
 
     if (!isDuplicate) {
+      ServerStoreImpl store = ehcacheStateService.getStore(name);
```

Comment:
So that is were I originally had the code, but the method never seemed to get called in my testing.  Although after talking with you yesterday now I know why it wasn't called.  I do agree the placement is better here.  I just made the change and tested it.  It seems to be working fine.